contrib/update-lock-files.sh

```diff
@@ -6,6 +6,13 @@ set -euo pipefail
 
 for file in Cargo-minimal.lock Cargo-recent.lock; do
     cp --force "$file" Cargo.lock
-    cargo check --all-features
+    # Build each crate individually to avoid conflicting features
+    (cd client && cargo check --all-features)
+    (cd types && cargo check --all-features)
+    (cd jsonrpc && cargo check --all-features)
+    (cd node && cargo check --features=29_0)
+    # verify crate is excluded from workspace, handle separately
+    (cd verify && cargo check)
+    # Skip integration_test as it cannot use --all-features
```

Comment:
`integration_test` can be checked the same as `node` (and for the same reason).

What about using lists of crate names and then a for loop, might be cleaner? I don't see the reason to have `verify` separate.